q2lint.py

```diff
@@ -27,6 +28,19 @@ def main():
             if header != HEADER:
                 errors.append('Invalid header: %s' % filepath)
 
+    npm_packages = filter(lambda x: 'node_modules' not in x,
+                          glob.glob('**/*/package.json', recursive=True))
+    if npm_packages:
+        import subprocess
+        import os
+
+        for package in npm_packages:
+            pkg_path, _ = os.path.split(package)
+            cmd = 'cd %s; npm i; npm run build; git diff --quiet */bundle.js' \
+                  ' || bash -c "exit -1"; git checkout -- .' % pkg_path
+            if subprocess.run(cmd, shell=True).returncode != 0:
+                errors.append('npm build error on %s' % pkg_path)
```

Comment:
Is this the message devs will see if the build is outdated and needs a rebuild? If so, it might be nice to provide two different error messages: one indicating a rebuild is necessary, and one indicating that the subprocess command(s) failed. It'd also be handy to provide stdout/stderr for the latter case.